cr-examples/onnx/src/main/java/oracle/code/onnx/OnnxProtoBuilder.java

```diff
@@ -260,25 +261,53 @@ <P> T forEach(Iterable<P> sup, BiConsumer<T, ? super P> cons) {
     static final int IR_VERSION = 10;
     static final int OPSET_VERSION = 21;
 
+    private static final class Indexer extends IdentityHashMap<Value, String> {
```

Comment:
You can use a HashMap, since the expression `v1.equals(v2) == (v1 == v2)` always returns true. This applies to all code item instances.